kratos/geometries/brep_curve_on_surface.h

```diff
@@ -114,6 +116,7 @@ class BrepCurveOnSurface
         bool SameCurveDirection = true)
         : BaseType(PointsArrayType(), &msGeometryData)
         , mpCurveOnSurface(pNurbsCurveOnSurface)
+        , mCurveNurbsInterval(pCurve->DomainInterval())
```

Comment:
I think you changed the wrong constructor.

Reply:
thanks, it was the right constructor, but a wrong change. ;)